spreadsheets.qmd

```diff
@@ -280,63 +278,37 @@ deaths
 ```
 
 The top three rows and the bottom four rows are not part of the data frame.
-
```

Comment:
BTW I removed this just in the interests of space and because it didn't seem that important.